client/src/components/MealCard.js

```diff
@@ -0,0 +1,127 @@
+// WILL MOVE ADD TO CART BUTTON ONCE CHEF PROFILE PAGE IS CREATED
+import React, { useContext } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import {
+    Card,
+    Divider,
+    Avatar,
+    CardActions,
+    CardContent,
+    CardMedia,
+    Button,
+    Typography,
+    Grid,
+} from "@material-ui/core";
+import { theme } from "../themes/theme";
+import { CartContext } from "../contexts/cart/CartContextProvider";
+import { dollarFormatter } from "../lib/utils";
+
+const useStyles = makeStyles({
+    root: {
+        maxWidth: 225,
+        borderRadius: "0px",
+        boxShadow: "0px 0px 10px 5px rgba(7,7,7,0.05)",
+        margin: theme.spacing(3),
+    },
+    small: {
+        marginTop: theme.spacing(0.5),
+        marginBottom: theme.spacing(0.5),
+        marginLeft: theme.spacing(1),
+        width: theme.spacing(4),
+        height: theme.spacing(4),
+    },
+    subtitle2: {
+        fontSize: ".3rem",
+        color: theme.palette.grey[600],
+    },
+    subtitle1: {
+        fontWeight: "bold",
+        fontSize: ".7rem",
+    },
+    subtitle: {
+        fontWeight: "bold",
+    },
+});
+
+function MealCard(props) {
+    const classes = useStyles();
+    const { chef, addToCart } = useContext(CartContext);
+    const { mealPic, title, price, chefName, chefPic, location, id, chefId } = props;
+
+    const addMeal = (e) => {
+        e.preventDefault();
+        const id = parseFloat(e.currentTarget.value);
```

Comment:
Since this component only has the one id passed by props, we shouldn't need this